modules/oe_webtools_analytics/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenEuropa Webtools Analytics
+
+A Webtools Laco service that provides information on page access to a 3rd party analytics service.
+
+### How it works
+
+This module triggers an event on a page load that allows subscribers to decide if the particular
+route is going to be tracked or not. It also allows to define all the available variables to customize the information being send to the analytics service.
+You can find more information on the available parameters [here](https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Piwik).
```

Comment:
This link has restricted access. We can provide this link instead:
http://ec.europa.eu/ipg/services/analytics/
In this page the current link is available (Documentation), so information will be complete.